### DISCUSSION TITLE
为何不支持多个key的填写啊?

### DISCUSSION THREAD

**Discussion by buwanyuanshen** [lobehub/lobehub#1235] [General Question]
快快修改完善一下！！！还有自定义初始模型，环境变量添加gemini pro(gemini key和url)的使用。哈哈哈哈😄，提的有点多了。

**Comment by buwanyuanshen** [comment]
还有就是运行一次5分钟左右太慢了吧？其他聊天web大多数30–60秒就部署完了。

**Reply by arvinxx** [reply]
最早一版本也是一分钟部署完的，但现在功能特性这么多，依赖也多，自然就慢了。

**Comment by arvinxx** [comment]
PR welcome
